src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -4328,11 +4349,28 @@ export class ResultsViewPageStore
         {
             await: () => [this.studyIds],
             invoke: async () => {
-                return client.fetchMolecularProfilesUsingPOST({
+                let profiles = await client.fetchMolecularProfilesUsingPOST({
                     molecularProfileFilter: {
                         studyIds: this.studyIds.result!,
                     } as MolecularProfileFilter,
                 });
+
+                // expression profiles are not allowed
+                // under some circumstances
+                if (allowExpressionProfiles(this.studies.result)) {
+                    return profiles;
+                } else {
+                    return profiles.filter(
+                        p =>
+                            ![
+                                AlterationTypeConstants.MRNA_EXPRESSION,
```

Comment:
@inodb i think it's best to exclude rather than include because if we introduce new profiles they will inadvertantly be excluded.  Of course, this is temporary.  If we're serious about this we actually need to model this in data.

Reply:
sounds good